src/aosm/azext_aosm/aaz/latest/aosm/publisher/artifact_manifest/_update_state.py

```diff
@@ -0,0 +1,238 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "aosm publisher artifact-manifest update-state",
+    is_preview=True,
+)
+class UpdateState(AAZCommand):
+    """Update state for artifact manifest.
```

Comment:
Please add examples for those aaz commands. You can reference this [doc](https://azure.github.io/aaz-dev-tools/pages/usage/workspace-editor/#command-examples)

Reply:
Thank you and done.